bentoml/service/management.py

```diff
@@ -0,0 +1,267 @@
+import logging
+import os
+
+from bentoml.exceptions import BentoMLException
+from bentoml.utils import status_pb_to_error_code_and_message
+from bentoml.utils.lazy_loader import LazyLoader
+from bentoml.utils.usage_stats import (
+    track,
+    track_load_start,
+    track_load_finish,
+    track_save,
+)
+from bentoml.yatai.client import YataiClient
+from bentoml.yatai.yatai_service import get_yatai_service
+from bentoml.saved_bundle.loader import load_from_bundle_path, _is_remote_path
+
+logger = logging.getLogger(__name__)
+yatai_proto = LazyLoader('yatai_proto', globals(), 'bentoml.yatai.proto')
+
+
+def save(bento_service, yatai_url=None, version=None, labels=None):
+    """
+    Save and register the given BentoService via BentoML's built-in model management
+    system. BentoML by default keeps track of all the SavedBundle's files and metadata
+    in local file system under the $BENTOML_HOME(~/bentoml) directory. Users can also
+    configure BentoML to save their BentoService to a shared Database and cloud object
+    storage such as AWS S3.
+
+    :param bento_service: target BentoService instance to be saved
+    :param yatai_url: optional - URL path to yatai server
+    :param version: optional - save with version override
+    :param labels: optional - label dictionary
+    :return: saved_path: file path to where the BentoService is saved
+    """
+
+    if yatai_url:
+        yatai_service = get_yatai_service(channel_address=yatai_url)
+        yatai_client = YataiClient(yatai_service)
+    else:
+        yatai_client = YataiClient()
+
+    track_save(bento_service)
+    return yatai_client.repository.upload(bento_service, version, labels)
+
+
+def load(bento, yatai_url=None):
+    """
+    Load a BentoService instance base on the BentoService tag (key:version) or path.
+
+    Args:
+        bento: a BentoService identifier in the format of NAME:VERSION or a path like.
+        yatai_url: optional. a YataiService URL address.
+    Returns:
+        BentoService instance
+    """
+    track_load_start()
+    if _is_remote_path(bento) or os.path.isdir(bento):
+        if yatai_url:
+            logger.info('Path to BentoService is provided, ignoring the yatai url.')
+        svc = load_from_bundle_path(bento)
+        track_load_finish(svc)
```

Comment:
I'd prefer moving this back to `load_from_bundle_path` and avoid changing the semantic of this metric